.circleci/config.yml

```diff
@@ -1,140 +1,28 @@
 version: 2.1
-parameters:
```

Comment:
cleaned up CI as we are only generating content manually. once there is a better automated process - the config can be revert. otherwise - this is a dead code.